### PR TITLE
JBPM-6138: Add info icon with help content for string wildcards

### PR DESCRIPTION
**Pull request by nmirasch**
Show allowed string widlcards help when the filter includes any text filter

### CODE REVIEW

File: jbpm-wb-common/jbpm-wb-common-client/src/main/resources/org/jbpm/workbench/common/client/resources/i18n/Constants.properties [hunk 1]
```diff
@@ -22,4 +22,10 @@ ClearAll=Clear All
...
 To=To
 Yes=Yes
-No=No
\ No newline at end of file
+No=No
+AllowedWildcardsForStrings=The allowed wildcards for Strings are:
+AsubstituteForASingleCharacter=A substitute for a single character.
+ASubstituteForZeroOrMoreCharacters=A substitute for zero or more characters.
+Charlist=charlist
+SetsAndRangesOfCharactersToMatch=Sets and ranges of characters to match.
```
Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/resources/org/jbpm/workbench/common/client/resources/i18n/Constants.properties]: This sentence doesn't look clear enough.

Comment (cristianonicolai) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/resources/org/jbpm/workbench/common/client/resources/i18n/Constants.properties]: @bsremac perhaps: _Set of characters to match_ ?

Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/resources/org/jbpm/workbench/common/client/resources/i18n/Constants.properties]: Sounds much better to me :)

Comment (nmirasch) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/resources/org/jbpm/workbench/common/client/resources/i18n/Constants.properties]: @bsremac, @cristianonicolai changed!

**Comment by bsremac** [comment]
+1

**Pull request merged**